examples/AerosolActivation/aerosol_activation.jl

```diff
@@ -0,0 +1,275 @@
+#   and run the Julia package manager to download all the dependecies.
+
+# ## Example
+
+# We begin by importing some standard Julia modules,
+# the Ensemble Kalman Process modules,
+# CLIMA Parameter modules and Aerosol Activation modules.
+using Plots
+using Distributions
+using LinearAlgebra
+using Test
+using Random
+rng_seed = 44
+
+import EnsembleKalmanProcesses
+const EKP = EnsembleKalmanProcesses
+
+import CLIMAParameters
+const CP =  CLIMAParameters
+struct EarthParameterSet <: CP.AbstractEarthParameterSet end
+const param_set = EarthParameterSet()
+
+import CloudMicrophysics
+const AM = CloudMicrophysics.AerosolModel
+const AA = CloudMicrophysics.AerosolActivation
+
+import Thermodynamics
+const TD = Thermodynamics
+
+# Next, we provide the information about the priors of the parameters we want to learn.
+# We are calibrating two parameters decribing the aerosol properties -
+# namely the aerosol molar mass and the osmotic coefficient.
+parameter_names = ["molar_mass", "osmotic_coeff"]
+
+# In this test we do know the parameter values.
+# We use them to test the convergence of EKP for aerosol activation.
+molar_mass_true = 0.058443
+osmotic_coeff_true = 0.9
+default_params = [molar_mass_true, osmotic_coeff_true]
+
+# Both parameters have to be positive definite, therefore we
+# define the constraints to be bounded below by zero.
+c1 = EKP.ParameterDistributionStorage.bounded_below(0.0)
+c2 = EKP.ParameterDistributionStorage.bounded_below(0.0)
+constraints = [[c1], [c2]]
+
+# We don't have any other prior knowledge about the calibrated parameters.
+# Defining the prior distribution in the constrained space to be normal
+# with the mean equal to zero and the standard deviation equal to one,
+# ensures that the default parameter values are well within the
+# assumed prior pdf.
+d1 = EKP.ParameterDistributionStorage.Parameterized(Distributions.Normal(0,1))
+d2 = EKP.ParameterDistributionStorage.Parameterized(Distributions.Normal(0,1))
+distributions = [d1, d2]
+
+# This concludes the setup of priors.
+priors = EKP.ParameterDistributionStorage.ParameterDistribution(distributions, constraints, parameter_names);
+
+# Next we define the atmospheric conditions for which the calibration will take place,
+# (air temperature in K, air pressure in Pa vertical velocity in m/s
+# and vapor specific humidity assuming its saturated in kg/kg)
+# This can be changed later to include more than one (T,p,w) combination in the calibartion process
+T = 283.15
+p = 100000.0
+w = 5.0
+p_vs = TD.saturation_vapor_pressure(param_set, T, TD.Liquid())
+q_vs = 1 / (1 - CP.Planet.molmass_ratio(param_set) * (p_vs - p) / p_vs)
+q = TD.PhasePartition(q_vs, 0.0, 0.0)
+
+# We also define the aerosol size distribution (lognormal, 1 mode)
+# with (mean radius in m, geometric stdev, number concentration 1/m3).
+# These can also be changed later to include different initial size distributions.
+r_dry = 0.243 * 1e-6
+stdev = 1.4
+N = 100.0 * 1e6
```

Comment:
```suggestion
N = 1e8
```

Reply:
I'll keep it like this. It's 100 per cm3. So I think it's nicer for the human eye to have the cm3 separated from the value

Reply:
```suggestion
N = 100.0 * 1e6 # since 1/cm³ = 1e6 1/m³ 
```